doc/cephfs/fs-nfs-exports.rst

```diff
@@ -17,29 +17,52 @@ Create NFS Ganesha Cluster
 
 .. code:: bash
 
-    $ ceph nfs cluster create <type=cephfs> [--size=1] <clusterid>
+    $ ceph nfs cluster create <type=cephfs> <clusterid> [<placement>]
```

Comment:
We need to restrict the cluster id names. I was able to accidentally create exports with cluster id as `/bar`.

Also: reject exports for clusters that do not exist.

Reply:
(And add tests for these cases.)

Reply:
I will do this change in last. Checking of clusters in exports would require changes in vstart too. As we are using test_orchestrator.